docs/site/Server.md

```diff
@@ -23,7 +23,7 @@ listening on different ports and working with different protocols.
 
 LoopBack 4 currently offers the
 [`@loopback/rest`](https://github.com/strongloop/loopback-next/tree/master/packages/rest)
-package out of the box which provides an HTTP based server implementation
+package out of the box which provides an HTTP/HTTPS based server implementation
```

Comment:
I know you didn't write this line but can we rephrase this line ... I had to read it like 10 times for. We don't offer `@loopback/rest` out of the "box" ... it's a standalone module. 

LoopBack 4 offers the `@loopback/rest` package which provides out of the box HTTP/HTTPS based server called `RestServer` for handling REST requests.